src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Editor/Listeners/FrameOpenCloseListener.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Threading;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Editor
+{
+    [Export(typeof(IFrameOpenCloseListener))]
+    internal class FrameOpenCloseListener : OnceInitializedOnceDisposedAsync, IFrameOpenCloseListener, IVsWindowFrameEvents, IVsSolutionEvents
+    {
+        private readonly IServiceProvider _serviceProvider;
+        private readonly IEditorStateModel _editorModel;
+        private readonly IProjectThreadingService _threadingService;
+        private readonly UnconfiguredProject _unconfiguredProject;
+        private IVsWindowFrame _frame;
+        private uint _eventCookie = VSConstants.VSCOOKIE_NIL;
+        private uint _solutionEventCookie = VSConstants.VSCOOKIE_NIL;
+
+        [ImportingConstructor]
+        public FrameOpenCloseListener(
+            [Import(typeof(SVsServiceProvider))]IServiceProvider helper,
+            IEditorStateModel editorModel,
+            IProjectThreadingService threadingService,
+            UnconfiguredProject unconfiguredProject) :
+#pragma warning disable IDE0030 // Use null propagation https://github.com/dotnet/roslyn/issues/16114
```

Comment:
I didn't want to suppress this error globally: it's a useful error and we should use null-propagation when it does the correct thing. It's just getting triggered incorrectly here.